src/Microsoft.AspNetCore.Testing/Tracing/EventSourceTestCollection.cs

```diff
@@ -0,0 +1,10 @@
+namespace Microsoft.AspNetCore.Testing.Tracing
+{
+    // This file has to be public and has to actually live INSIDE the test assembly. Hence it is a shared-source file.
+    // This file is NOT part of Microsoft.AspNetCore.Testing.Tracing's compilation sources, it is exported to NuGet brought in to each
+    // see xunit bug: [TODO]
+    [Xunit.CollectionDefinition("Microsoft.AspNetCore.Testing.Tracing.EventSourceTestCollection", DisableParallelization = true)]
+    public class EventSourceTestCollection
```

Comment:
This class is would be included both in `Microsoft.AspNetCore.Testing.Tracing` and user assembly, would it cause an issue?

Reply:
It shouldn't unless you try to use it. The assembly name will be different. I'm going to look at remove it though, it's not necessary that it be compiled in.